nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -318,3 +318,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The option `services.prometheus.exporters.pihole.interval` does not exist anymore and has been removed.
 
 - `k3s` can now be configured with an EnvironmentFile for its systemd service, allowing secrets to be provided without ending up in the Nix Store.
+
+- `gitea` module options have been made more RFC042 conforming (i.e. some options were removed in favor the official ones by using `settings`)
```

Comment:
```suggestion
- `gitea` module options have been changed to be RFC042 conforming (i.e. some options were moved to be located under `services.gitea.settings`)
```

otherwise people thing, that things would have been fully removed.
also please see the comment below the above headline.